mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/errorhandlers/payloadbuilders/MarcBibDiErrorPayloadBuilder.java

```diff
@@ -27,6 +27,7 @@
 @Component
 public class MarcBibDiErrorPayloadBuilder implements DiErrorPayloadBuilder {
   private static final String FIELDS = "fields";
+  public static final String NO_CONTENT_TITLE = "No content";
```

Comment:
I think this string exists somewhere else. It would be good to unify all seperate instances into a util or constants class, so that the no content title can be uniform across the app.